src/cmd/cli/command/commands.go

```diff
@@ -390,7 +400,7 @@ var RootCmd = &cobra.Command{
 				term.ResetWarnings() // clear any previous warnings so we don't show them again
 
 				defer func() { track.Cmd(nil, "Login", P("reason", err)) }()
-				if err = cli.InteractiveLogin(cmd.Context(), client, getCluster()); err != nil {
+				if err = cli.InteractiveLoginPrompt(cmd.Context(), client, getCluster(), port); err != nil {
```

Comment:
This now opens the browser without user interaction. Not a change we want, I think.